Add support for global attributes

ouun opened the pull request:
Allows to add global attributes to all SVGs which are overwritable.

ouun committed:
Add default 'attributes' to config file

ouun committed:
Merge global config and SVG attributes

ouun committed:
Merge pull request #1 from ouun/patch-1

Add default 'attributes' to config file

ouun committed:
Merge pull request #2 from ouun/patch-2

Merge global config and SVG attributes

ouun commented:
Dear @Log1x,
what do you think? Would that be useful? 

Kind regards

Log1x commented:
yeah this would definitely be sane to add – I'll try to push a release later today! sorry for the delay.

Log1x committed:
chore(config): Change array to shorthand, make language more uniform
enhance(sagesvg): Default attributes to array

Log1x committed:
chore(deps): Bump dependencies
chore(license): Bump license year
chore(deps): Bump minimum PHP version to match Acorn

Log1x committed:
chore(linting): Appeal to the phpcs gods

Log1x commented:
thanks a lot!
